src/components/oParl/previews/PersonPreview.tsx

```diff
@@ -4,26 +4,32 @@ import { texts } from '../../../config';
 
 import { getFullName } from '../../../helpers';
 import { PersonPreviewData } from '../../../types';
-import { OParlPreviewEntry } from './OParlPreviewEntry';
+import { TextListItem } from '../../TextListItem';
+import { getOrganizationNameString } from '../oParlHelpers';
 
 type Props = {
   data: PersonPreviewData;
   navigation: NavigationScreenProp<never>;
 };
 
 export const PersonPreview = ({ data, navigation }: Props) => {
-  const { id, type } = data;
+  const { id, membership, type } = data;
 
   const title = getFullName(texts.oparl.person.person, data);
+  const faction = membership?.find(
+    (membership) => membership.organization?.classification === 'Fraktion' && !membership.endDate
+  )?.organization;
 
-  return (
-    <OParlPreviewEntry
-      id={id}
-      type={type}
-      title={title}
-      topDivider={true}
-      navigation={navigation}
-      screenTitle={texts.oparl.person.person}
-    />
-  );
+  const params = { id, type, title: texts.oparl.person.person };
```

Comment:
`person.person`? could there be a different subkey for better readability? i have not checked the texts file to see myself, sorry for that.

Reply:
i had a look.

```
person: {
      affix: 'Namenszusatz:',
      body: 'Körperschaft',
      email: 'Email:',
      faction: 'Fraktion:',
      familyName: 'Familienname:',
      formOfAddress: 'Anrede:',
      gender: 'Geschlecht:',
      givenName: 'Vorname:',
      license: 'Lizenz: ',
      life: 'Leben: ',
      lifeSource: 'Quelle zu Lebensangaben: ',
      location: 'Ort',
      membership: 'Mitgliedschaften',
      name: 'Name:',
      person: 'Person',
      persons: 'Personen',
      phone: 'Telefon:',
      status: 'Status:',
      title: 'Titel:'
    },
```

should be fine. could think of `singular` and `plural` or something, but no need to change that, i would say.

Reply:
:ok: